<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M_append</class>
 <widget class="QWidget" name="M_append">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Append</string>
  </property>
  <widget class="QToolButton" name="toolButton_store">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>儲存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_phone">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>電話</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_memo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>備註</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>441</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_prev">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上一筆</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_next">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一筆</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 85, 255);font: 12pt &quot;AR PL UKai TW&quot;;</string>
   </property>
   <property name="text">
    <string>新增</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_index">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
